### ISSUE TITLE
About GLideN64 Window Don´t Show Translation

### ISSUE THREAD

**Issue by Predator82Germany** [gonetz/GLideN64#494]
![172](https://cloud.githubusercontent.com/assets/6679921/7445037/b9398fcc-f1a1-11e4-84f6-b1370630f8f0.jpg)

**Comment by Gillou68310** [comment]
@gonetz are you sure yongzh is the author of gles2n64? IIRC this plugin has been ported by the pandora team.

**Comment by gonetz** [comment]
Cite gles2n64\src\COPYING : 
"This directory contains the source code of gles2n64 ported to Android
by yongzh (freeman.yong@gmail.com). The original source code is available at:

http://code.google.com/p/gles2n64/"

**Comment by Gillou68310** [comment]
Porting to android doesn't necessary means that he ported it to gles ;-)

**Comment by gonetz** [comment]
Yep, you're right. If you know the real author of gles port, I'll add him to credits too.
